values/cert-manager/cert-manager-raw.gotmpl

```diff
@@ -71,20 +68,37 @@ resources:
                   key: secret
                   name: "dns-secret"
               {{- end }}
-      {{- end }}
-{{- if eq $clusterIssuer "selfsigned-issuer" }}
+{{- else }}
+  - apiVersion: cert-manager.io/v1
+    kind: ClusterIssuer
+    metadata:
+      name: {{ $clusterIssuerName }}
+    spec:
+      ca:
+        secretName: {{ $clusterIssuerName }}
+  {{- if eq $v.cluster.provider "kind" }}
   - apiVersion: v1
     kind: Secret
     metadata:
-      name: {{ $caKeyPair }}
+      name: {{ $clusterIssuerName }}
     data:
       tls.crt: {{ requiredEnv "KIND_CA_CERT" | b64enc }}
       tls.key: {{ requiredEnv "KIND_CA_KEY" | b64enc }}
+  {{- else }}
+  - apiVersion: v1
+    kind: Secret
+    metadata:
+      name: {{ $clusterIssuerName }}
+    type: kubernetes.io/tls
+    data:
+      tls.crt: {{ ($cm | get "customRootCA" "")| b64enc }}
+      tls.key: {{ ($cm | get "customRootCAKey" "") | b64enc}}
+  {{- end }}
 {{- end }}
 # generate all da certs
 {{- range $domInfo := $doms.domInfo }}
   {{- if not $domInfo.hasCert }}
     {{- $certName := ($domInfo.domain | replace "." "-") }}
-  - {{ tpl $certTpl (merge (dict "domain" $domInfo.domain "name" $certName "issuer" $clusterIssuer) $v) | nindent 4 }}
+  - {{ tpl $certTpl (dict "domain" $domInfo.domain "name" $certName "issuer" $cm.issuer "clusterIssuerName" $clusterIssuerName "hasCloudLB" $v.otomi.hasCloudLB ) | nindent 4 }}
```

Comment:
Why not pass on `otomi` in it's entirety? It provides context and avoids plucking separate props, and stays aligned with other parts of the code doing the same

Reply:
passing otomi for just one flag doesn't seem to be a good idea

Reply:
Well, there is no need to add more code plucking props, and I just gave good reasons to pass otomi as is:
- we want to have a generic approach to inject values throughout the code, instead of plucking them to flatten
- which leads to work (plucking) and maintenance that can be avoided
But I can refactor this as well while I am at it...

Reply:
yes, please do so